### PR TITLE
Add stock ticker

### PR DESCRIPTION
**Pull request by mok4ry**
Add stock ticker, including connection to Yahoo Finance API.

Also add new dependency, `classnames`.

### CODE REVIEW

File: app/js/actions/stock.js [hunk 1]
```diff
@@ -1,28 +1,57 @@
 export const ADD_STOCK = 'ADD_STOCK';
 export const UPDATE_STOCK = 'UPDATE_STOCK';
+export const GET_STOCK = 'GET_STOCK';
```
Comment (vuzDylan) [inline app/js/actions/stock.js]: Were is this used?

File: app/js/containers/StockTicker.jsx [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+
+    this.state = {
+      tickers : [
+        'AAPL',
+        'MSFT',
+        'TSLA',
+        'CMRX'
+      ]
+    };
+  }
```
Comment (vuzDylan) [inline app/js/containers/StockTicker.jsx]: Does this need state it seems to be unchanging

File: app/js/reducers/stock.js [hunk 3]
```diff
@@ -1,12 +1,11 @@
 import { ADD_STOCK, UPDATE_STOCK } from '../actions/stock';
```
Comment (vuzDylan) [inline app/js/reducers/stock.js]: If update isn't used we should remove it

File: app/css/components/StockTickerItem.css [hunk 4]
```diff
@@ -0,0 +1,17 @@
...
+  margin-right: 5px;
+}
+
+.stock-ticker-item > .sti-change-positive {
+  color: #00FF00;
+}
+
+.stock-ticker-item > .sti-change-negative {
+  color: #FF0000;
+}
```
Comment (kocsenc) [inline app/css/components/StockTickerItem.css:17]: I'm assuming `broswerify` includes this automagically.

Comment (kocsenc) [inline app/css/components/StockTickerItem.css:17]: `gulp` does dat. GG

Comment (mok4ry) [inline app/css/components/StockTickerItem.css:17]: Yeah, `gulp` is configured to recursively find and concatenate css files:
https://github.com/TeamMartinez/YOLO-Client/blob/stock-ticker/gulp/config.js#L13

File: package.json [hunk 5]
```diff
@@ -6,6 +6,7 @@
     "start": "node ./bin/www"
   },
   "dependencies": {
+    "classnames": "^2.2.3",
```
Comment (kocsenc) [inline package.json:9]: wut?

File: app/js/containers/StockTicker.jsx [hunk 6]
```diff
@@ -0,0 +1,36 @@
...
+import StockTickerItem from '../components/StockTickerItem';
+import { getStocks } from '../actions/stock';
+
+function mapStateToProps(state) {
+  return {
+    stocks: state.stocks
+  }
+}
+
+const INITIAL_DATA = ['AAPL', 'MSFT', 'TSLA', 'CMRX'];
```
Comment (kocsenc) [inline app/js/containers/StockTicker.jsx:14]: ~~Why are we not using this as default initial data?~~ When I ran it it only showed AAPL, I would assume we want to show all of these right?

**Comment by kocsenc** [comment]
LGTM

File: app/js/containers/StockTicker.jsx (hunk 6 above)
Comment (jonobrien) [inline app/js/containers/StockTicker.jsx:14]: pretty sure we want to show all the things.

Comment (mok4ry) [inline app/js/containers/StockTicker.jsx:14]: Should show all of them. Can you check console for errors?

Comment (kocsenc) [inline app/js/containers/StockTicker.jsx:14]: It's good to go, it just worked for me magically.

**Pull request merged**